src/components/ScrollSync/ScrollSyncNode.tsx

```diff
@@ -46,7 +46,8 @@ const ScrollSyncNode: React.ForwardRefExoticComponent<ScrollSyncNodeProps &
 
     const { registerNode, unregisterNode, onScroll } = useContext(ScrollingSyncerContext);
 
-    const ref = useRef<EventTarget & HTMLElement>(null);
+    const childRef = (children as any).ref;
```

Comment:
I had to do an ugly cast to `any` here, because `React.ReactElement` does not have the `ref` property in the types, and there is an error thrown if you try to access it from `props`.

I tried to find a better way of getting the ref, but I could not find anything.

Reply:
Yeah I see, no worries 👍🏽